tests/integration/workflows/test_authors_workflow.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2018 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""Tests for authors workflows."""
+
+from __future__ import absolute_import, division, print_function
+
+import pytest
+
+from invenio_workflows import (
+    ObjectStatus,
+    start,
+    workflow_object_class,
+)
+
+from jsonschema import ValidationError
+
+
+def test_authors_workflow_stops_when_record_is_not_valid(workflow_app):
```

Comment:
`s/workflow_app/isolated_app/`, because we want to achieve more test independence.